src/components/ContactList/ContactList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import './ContactList.css';
+import Contact from '../Contact';
+
+function ContactList() {
+  const contacts = [
+    { id: 1, link: '+375293837184', text: 'Contact us by Phone', to: 'tel:' },
+    {
+      id: 2,
+      link: '375293837184',
+      text: 'Contact us by WhatsApp',
+      to: 'https://api.whatsapp.com/send?phone=',
+    },
+    { id: 3, link: 'peterbabukh', text: 'Contact us by Skype', to: 'skype:' },
+    {
+      id: 4,
+      link: 'babukhpeter3@gmail.com',
+      text: 'Write us an E-mail',
+      to: 'mailto:',
+    },
+  ];
+
+  return (
+    <ul className="contact-list">
+      {contacts.map(el => (
+        <Contact key={el.id} link={el.link} text={el.text} to={el.to} />
```

Comment:
never use such simple id for keys. it should be unique. The chance that you can have another id={1} is very high.

Better reuse classnames notation:
``key={`contacts-contact-${contact.id}`}``